.spelling

```diff
@@ -0,0 +1,147 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+AlertManager
+Ansible
+CentOS
+CloudFront
+CockroachDB
+Conda
+CoreOS
+Couchbase
+Crunchbase
+D2iQ
+Datastax
+Dex
+Dispatchfile
+Dockerfile
+ESXi
+ElasticSearch
+Flagger
+Flink
+GitOps
+Github
+Golang
+Grafana
+HAProxy
+Hadoop
+Hashicorp
+Horovod
+Iinc
+InfluxData
+Istio
+Jupyter
+JupyterHub
+JupyterLab
+K8s
+Katib
+Kerberos
+Kibana
+Knative
+Kommander
+Konvoy
+Konvoy
+Kubeflow
+Kubernetes
+Linkerd
+Logstash
+MXNet
+MacBook
+MacOS
+Mesos
+MongoDB
+MySQL
+Nagios
+Nginx
+NiFi
+OneLogin
+ReLU
+OpenID
+OpenResty
+Percona
+PerconaXtraDB
+PipelineRuns
+PostgreSQL
+PyTorch
+Segment.io
+Splunk
+Tekton
+Telegraf
+TensorFlow
+TitanDB
+Toree
+VMWare
+VMware
+VirtualService
+XtraDB
+addon
+artifact
+artifacts
+autoscaler
+config
+configs
+hostname
+kubeaddon
+kubeaddons
+lifecycle
+mTLS
+plugin
+spec
+hyperparameter
+hyperparameters
+serverless
+SQLite
+KFServing
+MLOps
+MinIO
+MutatingWebhookConfiguration
+base64
+gRPC
+
+#
+# Stuff needed to make Kubeflow error free
+#
+28x28
+3rd-party
+ConfigMaps
+Eco-system
+# Should be ElasticSearch?
+Elasticsearch
+Onboarding
+PodDefaults
+PyTorchJob
+ResourceQuotaSpec
+S3-compliant
+TFJob
+U.S.
+a.k.a.
+additionalMetricNames
+alice
+args
+autoscaling
+base-64-encode
+# consensus?
+concensus
+containerd
+entrypoint
+funtions
+macOS
+mnist.py
+namespaced
+objectiveMetricName
+onboarding
+permissioned
+sigmoidal
+softmax
+stdout
+u
```

Comment:
this is not a spelling error?

Reply:
yeah, i was wondering about that one as well. i'm inclined to believe that this is either an entry in some table or a stray `u` that's used on some page (which is not a valid word obviously). 

the way to go forward would be to delete it from the dictionary and then use the interactive mode mentioned above to correct offences.

Reply:
thanks for the review btw!